cfg/cis-1.5/node.yaml

```diff
@@ -433,32 +433,6 @@ groups:
           systemctl daemon-reload
           systemctl restart kubelet.service
         scored: true
-
-      - id: 4.2.12
-        text: "Ensure that the RotateKubeletServerCertificate argument is set to true (Scored)"
```

Comment:
also here.

Reply:
Thanks. Tken care of here - https://github.com/aquasecurity/kube-bench/pull/1685